gost94/src/gost94.rs

```diff
@@ -15,10 +15,10 @@ pub type Block = GenericArray<u8, U32>;
 
 fn sbox(a: u32, s: &SBox) -> u32 {
     let mut v = 0;
-    for i in 0..8 {
+    for (i, item) in s.iter().enumerate().take(8) {
```

Comment:
I believe explicit loop over `i` conveys intent much better in such cases and has better readability, so I think it's better to reverse such changes.

Reply:
No problem